cross/minio/PLIST

```diff
@@ -0,0 +1 @@
+rsc:bin/minio
```

Comment:
It should be "bin" so that executable is stripped (removal of debug symbols) - I have no idea if "bin" is relevant for a Go binary.

Reply:
Fixed in https://github.com/SynoCommunity/spksrc/pull/3689/commits/119fdfe69ab441c3d285e6efe4e86407da5b68d9.